### ISSUE TITLE
GUI and screen reader accessibility

### ISSUE THREAD

**Issue by RealAmethyst** [VenimK/Subtitle-Forge#7]
The GUI does not have support for accessibility software like screen readers. I am fully blind and want to use this to convert picture based subtitles to srt so that I can use a TTS script I have to get them spoken out loud, however this GUI does not work with my screen reader, so I am unable to use it. Could this please be looked into?
